Library/.rubocop.yml

```diff
@@ -8,7 +8,15 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+
 Homebrew/CorrectBottleBlock:
+  Include:
+    - '**/Taps/homebrew/**/*.rb'
```

Comment:
Let's remove these includes and leave the logic in the cops themselves unless there's a way to do this that avoids duplicating this line for every cop.